index.bs

```diff
@@ -382,30 +379,32 @@ Magnetometers can be used for a variety of use-cases, such as:
   The user makes coarse gestures in the 3D space around the device using the magnet. Movement of the magnet affects the magnetic field sensed by the compass sensor integrated in the device.
   The temporal pattern of the gesture is then used as a basis for sending different interaction commands to the mobile device. Zooming, turning pages, accepting/rejecting calls, clicking items
   are some of the use cases.
-- In cases, where it is not possible to use orientation sensor, user may need to fuse magnetometer data with with other low-level sensor data, to calculate orientation.
-- Indoor navigation system may use magnetometer on mobile phones [[MAGINDOORPOS]]. Specific use cases for indoor navigation may include proximity advertising, way finding in malls or airports, geofencing, etc.
-- Calculating compass heading, more on that in [[#compass]].
+- A common use-case for magnetometers is sensor fusion in order to generate an Orientation Sensor which is stationary to the Earth plane, or a compass, which is basically the former with corrections to the declination depending on geolocation position, such that it points to the true north.
```

Comment:
Do you want to link to the Orientation Sensor spec (https://www.w3.org/TR/orientation-sensor/) , AbsoluteOrientationSensor to be specific, denoting that if sensor hub/hardware is unable to provide us and we use sensor fusion ourselves.